_posts/01-06-01-Vagrant.md

```diff
@@ -11,12 +11,13 @@ différents environnements à jour avec les mêmes versions pour l'ensemble des
 travaillez en équipe.
 
 Si vous développez sur Windows et que vous déployez votre code sur Linux (ou n'importe quoi qui ne soit pas Windows) 
-ou que vous travaillez en équipe, vous devriez penser à utiliser une machine virtuelle. Cela peut paraître compliqué mais 
-utiliser [Vagrant][vagrant] vous permet de mettre en place une machine virtuelle en seulement quelques étapes. 
+ou que vous travaillez en équipe, vous devriez penser à utiliser une machine virtuelle (Virtual Machine, abrégé en VM
+en anglais). Cela peut paraître compliqué mais utiliser [Vagrant][vagrant] vous permet de mettre en place une machine
+virtuelle en seulement quelques étapes. 
 Ces systèmes de base peuvent ensuite être configurés manuellement ou via des outils comme [Puppet][puppet] ou 
-[Chef][chef]. Configurer ces systèmes de façon automatisé est un bon moyen de s'assurer que les différents systèmes 
+[Chef][chef]. Configurer ces systèmes de façon automatisée est un bon moyen de s'assurer que les différents systèmes 
 mis en place seront configurés de la même manière sans avoir à maintenir une liste de commandes pour l'installation. 
-Vous pouvez aussi "détruire" votre système et en recréez un nouveau de façon entièrement automatisé, ce qui facilite 
+Vous pouvez aussi "détruire" votre système et en recréer un nouveau de façon entièrement automatisée, ce qui facilite 
```

Comment:
OK pour l'infinitif, par contre voir la note sur la conjugaison [plus haut](https://github.com/eilgin/php-the-right-way/pull/21/files#r101235766)